### ISSUE TITLE
Unexpected top-level property "name" error when using FlatCompat

### ISSUE THREAD

**Issue by C4illin** [schoero/eslint-plugin-better-tailwindcss#104]
```
ESLint: 9.28.0

Error: ESLint configuration in  » plugin:better-tailwindcss/recommended-warn is invalid:
        - Unexpected top-level property "name".
```

eslint.config.ts:

```typescript
import { FlatCompat } from "@eslint/eslintrc";
import js from "@eslint/js";
import type { Linter } from "eslint";
import { globalIgnores } from "eslint/config";

const compat = new FlatCompat({
  baseDirectory: import.meta.dirname,
});

export default [
  ...compat.config({
    parser: "@typescript-eslint/parser",
    extends: [
      "plugin:@typescript-eslint/recommended",
      "plugin:better-tailwindcss/recommended-warn",
      "next/core-web-vitals",
      "next/typescript",
    ],
    overrides: [
      {
        files: ["*.ts", "*.tsx", "*.js"],
        parser: "@typescript-eslint/parser",
      },
    ],
    parserOptions: {
      ecmaFeatures: {
        jsx: true,
      },
      ecmaVersion: "latest",
    },
    plugins: ["@typescript-eslint", "better-tailwindcss"],
    rules: {
      "better-tailwindcss/multiline": [
        "warn",
        {
          group: "newLine",
          printWidth: 100,
        },
      ],
    },
  }),
  js.configs.recommended,
  globalIgnores([".next/*", "node_modules/*"]),
] as Linter.Config[];
```

I got it to work when not using FlatCompat but that is needed for nextjs eslint
